api/database/migrations/data/2024_04_01_222259_create_draft_lyrics_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::connection('data')->create('draft_lyrics', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->uuid('track_id')->unique();
```

Comment:
Foreign key constraints?

Reply:
We have not added Foreign key constraints to any other table so far, do you still want me to apply to this table ?
If so, would it be similar to the below ?
```
$table->foreign('track_id')->references('id')->on('tracks');
```

Reply:
If we're not doing it anywhere else, I have no problem with not doing it here as well.